pyo3-ffi/src/cpython/longobject.rs

```diff
@@ -26,7 +26,8 @@ pub const Py_ASNATIVEBYTES_UNSIGNED_BUFFER: c_int = 4;
 pub const Py_ASNATIVEBYTES_REJECT_NEGATIVE: c_int = 8;
 
 extern "C" {
-    // skipped _PyLong_Sign
+    #[cfg(not(PyPy))]
+    pub fn _PyLong_Sign(v: *mut PyObject) -> c_int;
```

Comment:
We are generally moving towards not exposing any `_Py` prefixed functions, as they are a compatibility hazard.

As this is, could this please be added instead as `PyLong_IsPositive` / `PyLong_IsZero` / `PyLong_IsNegative` as per https://github.com/python/cpython/pull/126065?

If it's needed before Python 3.14 we should add these functions to `pyo3-ffi/src/compat`, which can use `_PyLong_Sign` as `pub (crate)` implementation detail.